### ISSUE TITLE
Can the search result scroll into view?

### ISSUE THREAD

**Issue by jb222** [Tampermonkey/tampermonkey#1156]
When clicking "Find Next" search results stay almost hidden in the bottom of the screen...
Could they scroll into view?

![search](https://user-images.githubusercontent.com/61352184/108841579-453ddd00-75d8-11eb-9203-e69a97b484cb.png)

**Comment by jb222** [comment]
Just bringing this back to your attention...

**Comment by jb222** [comment]
Once again reminding you of this issue (we're at 4.16 now)...

**Comment by wojnarw** [comment]
It's very irritating and should be easy to fix.

**Comment by Heistergand** [comment]
I totally agree this is irritating. The search bar should simply not overlay the code window. It instead should dock to it.

**Comment by derjanb** [comment]
Should be fixed at 4.19.6179 ([crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.
